Looks like that `ceph` is broken on `master`.

```bash
   46 |     IOOptions io_opts;
      |               ^~~~~~~
[100%] Linking CXX static library librocksdb.a
[100%] Built target rocksdb
[ 25%] Performing install step for 'rocksdb_ext'
[ 25%] Completed 'rocksdb_ext'
[ 25%] Built target rocksdb_ext
make: *** [Makefile:146: all] Error 2
error: builder for '/nix/store/3r9k3a8y1c03ylgy8a6fz9gw7hx00ylh-ceph-16.2.7.drv' failed with exit code 2;
       last 10 log lines:
       > /build/ceph-16.2.7/src/rocksdb/env/composite_env_wrapper.h:48:25: note: by argument 3 of type 'const rocksdb::IOOptions&' to 'rocksdb::IOStatus (rocksdb::FSSequentialFile::)(size_t, const rocksdb::IOOptions&, rocksdb::Slice*, char*, rocksdb::IODebugContext*)' {aka 'rocksdb::IOStatus (rocksdb::FSSequentialFile::)(long unsigned int, const rocksdb::IOOptions&, rocksdb::Slice*, char*, rocksdb::IODebugContext*)'}
       > /build/ceph-16.2.7/src/rocksdb/env/composite_env_wrapper.h:46:15: note: 'io_opts' declared here
       >    46 |     IOOptions io_opts;
       >       |               ^~~~~~~
       > [100%] Linking CXX static library librocksdb.a
       > [100%] Built target rocksdb
       > [ 25%] Performing install step for 'rocksdb_ext'
       > [ 25%] Completed 'rocksdb_ext'
       > [ 25%] Built target rocksdb_ext
       > make: *** [Makefile:146: all] Error 2
       For full logs, run 'nix log /nix/store/3r9k3a8y1c03ylgy8a6fz9gw7hx00ylh-ceph-16.2.7.drv'.
```

--- a/pkgs/development/python-modules/webtest/default.nix
+++ b/pkgs/development/python-modules/webtest/default.nix
@@ -1,55 +1,54 @@
 { lib
+, beautifulsoup4
 , buildPythonPackage
 , fetchPypi
-, isPy27
-, webob
-, six
-, beautifulsoup4
-, waitress
-, pyquery
-, wsgiproxy2
 , pastedeploy
+, pyquery
 , pytestCheckHook
+, pythonOlder
+, six
+, waitress
+, webob
+, wsgiproxy2
 }:
 
 buildPythonPackage rec {
-  version = "3.0.0";
   pname = "webtest";
-  disabled = isPy27; # paste.deploy is not longer a valid import
+  version = "3.0.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     pname = "WebTest";
     inherit version;
-    sha256 = "54bd969725838d9861a9fa27f8d971f79d275d94ae255f5c501f53bb6d9929eb";
+    hash = "sha256-VL2WlyWDjZhhqfon+Nlx950nXZSuJV9cUB9Tu22ZKes=";
   };
 
-  postPatch = ''
-    substituteInPlace setup.py --replace "nose<1.3.0" "nose"
-  '';
-
   propagatedBuildInputs = [
-    webob
-    six
     beautifulsoup4
+    six
     waitress
+    webob
   ];
 
   checkInputs = [
-    pytestCheckHook
     pastedeploy
-    wsgiproxy2
     pyquery
+    pytestCheckHook
+    wsgiproxy2
   ];
 
-  # Some of the tests use localhost networking.
   __darwinAllowLocalNetworking = true;
 
-  pythonImportsCheck = [ "webtest" ];
+  pythonImportsCheck = [
+    "webtest"
+  ];
 
   meta = with lib; {
     description = "Helper to test WSGI applications";
-    homepage = "https://webtest.readthedocs.org/en/latest/";
+    homepage = "https://webtest.readthedocs.org/";
     license = licenses.mit;
-    maintainers = with maintainers; [ ];
+    maintainers = with maintainers; [ fab ];
   };
 }

